src/bp-search/plugins/search-cpt/index.php

```diff
@@ -57,11 +57,11 @@ function bp_search_option_cpt_search( $items_to_search ) {
 function bp_search_helpers_cpts( $helpers ) {
 
 	$post_types          = get_post_types( array( 'public' => true ) );
-	$custom_handler_cpts = array( 'post', 'forum', 'topic', 'reply' );
+	$custom_handler_cpts = array( 'post', 'forum', 'topic', 'reply', 'post', 'page' );
```

Comment:
```suggestion
	$custom_handler_cpts = array( 'post', 'forum', 'topic', 'reply', 'page' );
```

@dipskakadiya Remove duplicate `post`